diff --git a/dist/util/git/config.js b/dist/util/git/config.js
index a6c3cab5fc156954c3c99d029b7fb2e7381bf4c6..96c8acefcbd06688719528e74c148296dcae3ba9 100644
--- a/dist/util/git/config.js
+++ b/dist/util/git/config.js
@@ -18,6 +18,7 @@ function simpleGitConfig() {
 	};
 	if (getEnv().RENOVATE_X_CLEAR_HOOKS) unsafe.allowUnsafeHooksPath = true;
 	const config = {
+		allowEnvironment: ["GIT_SSH_COMMAND"],
 		completion: {
 			onClose: true,
 			onExit: false
